docs/process/release.md

```diff
@@ -6,37 +6,48 @@ Example [Milestone](https://github.com/unicode-org/icu4x/milestone/5) and [Issue
-* Run `cargo update` for each `Cargo.lock` file to update our CI to freshest dependencies
-* Verify that the milestone and checklist are complete
-* Verify with component owners that they're ready for release
-* Verify that `ffi/diplomat` depends on a released (not Git) version of Diplomat. Get it published (ask manishearth or sffc) otherwise.
-* Verify that we have acceptable FFI coverage (should be a part of the checklist issue)
-  * Verify that `ffi/diplomat/tests/missing_apis.txt` is empty. If not, check with the team that we are okay punting these to the next release.
-* Take a bird-eye view at:
-  * READMEs
-  * Documentation
-  * Coverage
-  * Performance / Memory / Size benchmarks
-  * Cargo.toml files
-    * Cargo.toml files need to specify versions for each entry in `dependencies`.
-    * Cargo.toml should not specify versions for each entry in `dev-dependencies`.
-    * Ensure that any new packages have suitable version numbers.
-    * Ensure that the Cargo.toml file includes a set of fields consistent with other ICU4X packages.
+
+## Pre-release checklist
+
+This is a checklist of things that should be done in the weeks leading to the release.
+
+* [ ] Verify that the milestone and checklist are complete
+* [ ] Verify with component owners that they're ready for release
+* [ ] Take a bird-eye view at:
+  * [ ] READMEs
+  * [ ] Documentation
+  * [ ] Coverage
+  * [ ] Performance / Memory / Size benchmarks
+  * [ ] Cargo.toml files
+    * [ ] All dependencies from the workspace should use `workspace = true` rather than their version number or path 
+    * [ ] Cargo.toml files need to specify versions for each non-workspace entry in `dependencies`
+    * [ ] Cargo.toml should not specify versions for each workspace entry in `dev-dependencies`.
+    * [ ] Ensure that any new packages have suitable version numbers.
+    * [ ] Ensure that the Cargo.toml file includes a set of fields consistent with other ICU4X packages.
+* [ ] Run `cargo +nightly fmt -- --config=format_code_in_doc_comments=true --config=doc_comment_code_block_width=80` to prettify our docs
+* [ ] Run `cargo update` for each `Cargo.lock` file to update our CI to freshest dependencies
+* [ ] Go through `ffi/diplomat/tests/missing_apis.txt` and verify that it is empty. If it is not, component owners should either add FFI APIs, add `rust_link` annotations, or allowlist the relevant APIs as having been punted to the future
+* [ ] Verify that `ffi/diplomat` depends on a released (not Git) version of Diplomat. Get it published (ask manishearth or sffc) otherwise.
+* [ ] Start work on a changelog (see below)
+
+## Release steps
+
+Once the release checklist is complete, the assigned release driver will perform the following steps, in order:
+
+* Go through the prerelease checklist again, ensuring that no problems were reintroduced in the PRs that landed since the opening of the checklist. (Things like doc prettification will likely need to be rerun!)
```

Comment:
why rerun if we can just run it once? it's a lot of churn

Reply:
Ideally it will be a small amount of churn the second time. I think it's good to preflight as much as possible.

Reply:
Basically most of that checklist is going to need to be rerun anyway; because the whole point of this PR is to give people a bunch of parallelizable work that makes the final release smoother and less contentious, with the understanding that the parallelizeable work will happen over the course of a couple weeks and may miss changes that happen later.